cfgov/jinja2/v1/owning-a-home/closing-disclosure/closing-disclosure-1-data.html

```diff
@@ -0,0 +1,199 @@
+{% set data = {
+
+"img": url_for("static", filename="img/closing-disclosure-H25B-1.png"),
+
+"terms":
+    [
+        {
+            "term": "Check the spelling of your name",
+            "definition": "<p>Ask the lender to correct any inaccurate contact information. Even minor misspellings can cause big problems later.</p>",
+            "id": "check-name_1",
+            "category": "checklist",
+            "left": "44.68%",
+            "top": "11.95%",
+            "width": "19.86%",
+            "height": "4.50%"
+        },
+        {
+            "term": "Check that loan term, purpose, product, and loan type match your most recent Loan Estimate",
+            "definition": "<p>It's very important these items match what you were expecting. If they don't, call your lender immediately and ask why they have changed.</p>",
+            "id": "check-term_1",
+            "category": "checklist",
+            "left": "69.93%",
+            "top": "10.20%",
+            "width": "24.11%",
+            "height": "10.31%"
+        },
+        {
+            "term": "Check that the loan amount matches your most recent Loan Estimate",
+            "definition": "<p>If it has increased, ask your lender why. A possible reason could be that closing costs have been rolled into your loan. This reduces your upfront costs at closing, but adds to your overall costs because of the added interest you will pay.</p>",
+            "id": "loan-amount",
+            "category": "checklist",
+            "left": "5.96%",
+            "top": "28.07%",
+            "width": "88.23%",
+            "height": "3.07%"
+        },
+        {
+            "term": "Check your interest rate",
+            "definition": "<p>If your interest rate isn't what you were expecting, ask your lender why. If you locked your rate, your lender is only allowed to change it under limited circumstances.</p><p><a href=\"/askcfpb/143/whats-a-lock-in-or-a-rate-lock.html\" target=\"_blank\">Learn more about rate locks</a></p>",
```

Comment:
This is probably out of scope for this PR, but just noting that the Ask CFPB links throughout the explainer definitions use the old Ask URL structure & should be updated at some point so they don't require redirects.